packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

```diff
@@ -44,14 +44,21 @@ export class CloudinaryMediaStore implements MediaStore {
       throw new Error(responseData.message)
     }
     const fileRes = await res.json()
-
-    // TODO: be programmer
-    // NOTE: why do we need this?
     await new Promise((resolve) => {
```

Comment:
```suggestion
    /**
    * Images uploaded to Cloudinary aren't instantly available via the API;
    * waiting a couple seconds here seems to ensure they show up in the next fetch.
    */
    await new Promise((resolve) => {
```
Might be good to throw in an explainer now that we've decided to keep it in